WordPress/wordcamp.org
# CampTix: Improve ticket form accessibility

Fixes multiple instances of unlabelled inputs in the registration flow for screen reader users. Add headings & table structure hints to the available ticket table, and label for the quantity dropdown. Add & attach labels to the registration form inputs, including addons & custom fields. Wrap the checkboxes & radios into `fieldset`s, using [hidden labels](https://codepen.io/stevef/post/fielsdet-and-legend-semantics) to attach the question text.

Fixes https://meta.trac.wordpress.org/ticket/1591

### Screenshots

There should be no visual changes.

### How to test the changes in this Pull Request:

1. Go through the ticket buying process using a screen reader
2. Every input you hit should have an audible label

(alternately you can use dev tools to check the accessible name of each input)
